app/[locale]/admin/clients/page.tsx

```
@@ -265,182 +340,394 @@
-                          <p className="text-xs text-gray-500 dark:text-gray-500">
-                            {client.jobTitle}
+                <div key={client.id} className="px-6 py-4 hover:bg-gray-50 dark:hover:bg-gray-800/50 transition-colors">
+                                      <div className="flex items-center justify-between">
+                      <div className="flex items-center space-x-4 flex-1 min-w-0">
+                        <div className="flex items-center space-x-2 flex-shrink-0">
+                          <div className="w-4 h-4 bg-gray-200 dark:bg-gray-700 rounded"></div>
+                          <span className="text-sm text-gray-500 dark:text-gray-400">#{client.id.slice(0, 8)}</span>
+                        </div>
+                        <div 
+                          role="button"
+                          tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)